examples/hellocluster-yaml/config.yaml

```diff
@@ -0,0 +1,22 @@
+# The cluster to run on
+cluster: kislurm
```

Comment:
As a rule of thumb, let us not put identifying names from cluster in examples, those should be in private files. This one is of-course not revealing but we want to avoid leaking partitions and stuff like that.